.github/workflows/ci.yml

```diff
@@ -1,18 +1,18 @@
-name: Java CI
-
-on: [push, pull_request]
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK 17
-        uses: actions/setup-java@v2
-        with:
-          java-version: '17'
-          distribution: 'adopt'
-          cache: maven
-      - name: Build with Maven
-        run: mvn --batch-mode --update-snapshots verify
+name: Java CI

```

Comment:
It is weird that GitHub shows us that files were replaced with the same contents or changed names, but they did not, maybe you experimented with some branches or other things? Because it shouldn't be like that.
And it must have happened after my review, because it was Ok last time I saw your PR